pyproject.toml

```
@@ -39,7 +39,8 @@ dependencies = [
     # einops 0.3+ required for einops.repeat
     "einops>=0.3",
     # fiona 1.8.21+ required for Python 3.10 wheels
-    "fiona>=1.8.21",
+    # fiona 1.9.0+ required for fiona.listdir
+    "fiona>=1.9.0",
```

```suggestion
    # fiona 1.9+ required for fiona.listdir
    "fiona>=1.9",
```
Just to make things simpler